recipes/noqt/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version = "1.0" %}
+
+package:
+  name: noqt-helpers
+  version: {{ version }}
+
+
+build:
+  number: 0
+  noarch: generic
+
+outputs:
+  - name: noqt
+    requirements:
+      run:
+        - {{ pin_subpackage('noqt5', exact=True) }}
+        - {{ pin_subpackage('noqt6', exact=True) }}
+    test:
+      commands:
+        - echo "no tests for this package"
+  - name: noqt5
+    build:
+      noarch: generic
+    requirements:
+      run_constrained:
+        # qt-main only exists as of 5.x
+        - qt-main <5.0a0|>=6.0a0
```

Don't really need the `a0` for the `>=` condition.
```suggestion
        - qt-main <5.0a0|>=6.0
```